test/parallel/test-fs-make-callback.js

```diff
@@ -3,37 +3,31 @@ const common = require('../common');
 const assert = require('assert');
 const fs = require('fs');
 const cbTypeError = /^TypeError: "callback" argument must be a function$/;
+const callbackThrowValues = [null, true, false, 0, 1, 'foo', /foo/, [], {}];
 
-function test(cb) {
+function testMakeCallback(cb) {
   return function() {
     // fs.stat() calls makeCallback() on its second argument
-    fs.stat(__filename, cb);
+    fs.unlink(__filename, cb);
```

Comment:
Good point...but any of the operations which are calling `makeCallback` have an impact on the file. I would then use `fs.mkdtemp` which only creates a temp dir which we can delete at the end of the test. Does it make sense to you @lpinca?